pixart.js

```diff
@@ -1 +1,31 @@
+//* When I click the "Set Color" button, it should change the color of the "brush" box to the color I specify in the input field. (**Hint:** You will need to use `event.preventDefault()` somewhere in your code.)
```

Comment:
Hey John,

It looks like your first (or biggest) issue to fix is that you define 3 functions here, but they are never called, so nothing ever happens. 

I can't tell what your process was, but looking at your code, it looks like you kept trying to solve more problems without solving the first ones. In programming, order often matters, it's not worth moving forward until you've solved the first problem. Another way to say that is perhaps "always try to solve the simplest problem you can".

For example, instead of  
>When I click the "Set Color" button, it should change the color of the "brush" box to the color I specify in the input field.

solve:
> when I click the "Set Color" button, I should get an alert that pops up. (or console.log, whichever you prefer)
then
> when I click the "Set Color" button, I should get an alert that pops up with the text of what was typed in the box
then
> when I click the "Set Color" button, it should change the color of the brush box to "red"
then
> when I click the "Set Color" button, it should change the color of the brush box to whatever I typed in the box.